### DISCUSSION TITLE
Make this discussion an org-wise discussion

### DISCUSSION THREAD

**Discussion by tisonkun** [GreptimeTeam/greptimedb#3268] [General]
Check GitHub's doc - https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization

And the official discussion showcase - https://github.com/orgs/community/discussions

Greptime may not be that fruitful but we can still benefit for prioritizing the Discussions forum, making it a shared place to discuss Greptime technology and products.

We can keep the category structure AS IS for a while but focus on moving topic-based discussions here. Hopefully, the knowledge generated in topics is searchable and helps other community memebers.

cc @sunng87 @killme2008 @fengjiachun

**Comment by tisonkun** [comment]
I can see that this proposal is applied and I'd be glad to close it as resolved. However, there is no response and it's a bit confusing from my perspective. Would the admin takes the action leave a comment here so that we can track the history?

**Reply by evenyag** [reply]
I think it's fine to make it org-wise. By the way, do we have a guideline for what is preferred to be discussed in discussions first? We might still use issues to report bugs.

**Reply by tisonkun** [reply]
@killme2008 told me that he took the action.

> do we have a guideline for what is preferred to be discussed in discussions first

I'd try to find sometime update our [community](github.com/greptimeteam/community) page and put it into use as a document for how the Greptime community works.

We don't have to make the discussion process quite strict, but for how users find themselves conveniently:

* Issues for bug reports and concrete enhancements;
* Discussions for open-ended discussions and questions;
* [Slack](https://greptime.com/slack) for instant messaging.

**Comment by tisonkun** [comment]
Closed as accepted and done.
